exasol_script_languages_developer_sandbox/lib/export_vm/run_export_vm.py

```diff
@@ -32,17 +48,14 @@ def export_vm_image(aws_access: AwsAccess, vm_image_format: VmDiskImageFormat, t
     logging.info(
         f"Started export of vm image to {vm_bucket}/{bucket_prefix}. "
         f"Status message is {export_image_task.status_message}.")
-    last_progress = export_image_task.progress
-    last_status = export_image_task.status
+    last_progress = ExportImageTaskProgress.from_export_image_task(export_image_task)
     while export_image_task.is_active:
         time.sleep(config.global_config.time_to_wait_for_polling)
         export_image_task = aws_access.get_export_image_task(export_image_task_id)
-        if export_image_task.progress != last_progress or export_image_task.status != last_status:
-            logging.info(f"still running export of vm image to {vm_bucket}/{bucket_prefix}. "
-                         f"Status message is {export_image_task.status_message}. "
-                         f"Progess is '{export_image_task.progress}'")
-        last_progress = export_image_task.progress
-        last_status = export_image_task.status
+        progress = ExportImageTaskProgress.from_export_image_task(export_image_task)
+        if last_progress != progress:
+            logging.info(f"still running export of vm image to {vm_bucket}/{bucket_prefix}: {progress} ")
+            last_progress = progress
```

Comment:
to be honest, I would extract the whole while loop

Reply:
done